include/MySQL_HostGroups_Manager.h

```diff
@@ -261,6 +264,369 @@ class AWS_Aurora_Info {
 	~AWS_Aurora_Info();
 };
 
+struct p_hg_counter {
+	enum metric {
+		servers_table_version = 0,
+		server_connections_created,
+		server_connections_delayed,
+		server_connections_aborted,
+		client_connections_created,
+		client_connections_aborted,
+		com_autocommit,
+		com_autocommit_filtered,
+		com_rollback,
+		com_rollback_filtered,
+		com_backend_change_user,
+		com_backend_init_db,
+		// TODO: https://sysown-team.monday.com/boards/484798975/pulses/529413331
```

Comment:
We discussed this internally, please use a GH issue

Reply:
Yes, changing it was still in my TODO list.